modules/node-shared/src/main/scala/org/tessellation/node/shared/http/routes/SigningMessageRoutes.scala

```diff
@@ -0,0 +1,40 @@
+package org.tessellation.node.shared.http.routes
+
+import cats.effect.Async
+import cats.syntax.flatMap._
+import cats.syntax.option._
+
+import org.tessellation.node.shared.infrastructure.snapshot.services.SigningMessageService
+import org.tessellation.node.shared.infrastructure.snapshot.storage.SigningMessageStorage
+import org.tessellation.routes.internal._
+import org.tessellation.security.{Hasher, SecurityProvider}
+
+import eu.timepit.refined.auto._
+import org.http4s.HttpRoutes
+import org.http4s.dsl.Http4sDsl
+
+// TODO proper location/module for this class?
+// TODO proper name for this class?
```

Comment:
AdminFeeStakingSigningRoutes, MetagraphMessageRoutes, or some variation on this, also somewhat okay as is, but it might be ambiguous as is?